src/components/ConnectedHeaderbar.jsx

```diff
@@ -48,7 +54,21 @@ export function ConnectedHeaderBar({
             )
         }
         return params.appName
-    }, [appsInfoQuery.data, params])
+    }, [appsInfoQuery.data, params.appName])
+
+    // Set new displayname to page title when it updates
+    React.useEffect(() => {
+        if (appName) {
+            document.title = `${appName} | DHIS2`
+        }
+    }, [appName])
+
+    const appVersion = React.useMemo(() => {
+        if (!params.appName || !appsInfoQuery.data) {
+            return
+        }
+        return getAppVersion(params.appName, appsInfoQuery.data.apps)
+    }, [appsInfoQuery.data, params.appName])
```

Comment:
Some prep work for showing the version of the app (in addition to the global shell) in the header bar profile menu